css/mainresume.css

```diff
@@ -0,0 +1,46 @@
+.main-content {
+  position: absolute;
+  top: 8rem;
+  left: 10%;
+  width: 90%;
+}

```

Comment:
You are using this in conjunction with the class `row` - this defeats the purpose of using the grid system because you are setting your own widths and margins. This is Bootstrap's job!
This is why your content is not centered, because your sizing/spacing is clashing with Bootstrap's grid system. 
When applying grid classes to elements (`container`, `row`, `col-...`), you can add your own custom classes of course, but only for colours, text, **vertical** spacing etc - not for  horizontal spacing such as widths or left/right padding or margin.